addons/resource.language.en_gb/resources/strings.po

```diff
@@ -20024,7 +20024,16 @@ msgctxt "#36421"
 msgid "Bypassing VDPAU mixer saves resources on low power systems but slightly reduces picture quality."
 msgstr ""
 
-#empty strings from id 36422 to 36423
+#: system/settings/settings.xml
+msgctxt "#36422"
+msgid "Update trusted addons from"
+msgstr ""
+
+#. Description of setting with label #36422 "Update trusted addons from"
+#: system/settings/settings.xml
+msgctxt "#36423"
+msgid "By default Kodi will prevent any addons from trusted repositories being updated from private repositories. For cases such as updating from an addons beta repository this option can be switched to [Any repositories] (bear in mind this is a less secure option)."
```

Comment:
Might be best to remove the explicit mention of Kodi. A concerted effort has been done to simplify the ability for rebranders, so might be a good idea not to put it in a string.

Reply:
Maybe change the sentence to: `By default addons from trusted repositories will be prevented from being updated from private repositories.`

Reply:
done